### PR TITLE
[DOR-104] [SC] Problem with Service error page - Level 1

### PR DESCRIPTION
**Pull request by chandaksweta**

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -74,7 +74,8 @@ external.error.500.list-item2=Too many users on the service
 
 internal.error.500.title=There is a problem with the service
 internal.error.500.heading=Sorry, there is a problem with the service
-internal.error.500.message=Contact the Helpline.
+internal.error.500.line1=Try again later.
```
Comment (Tasmine10ten) [inline conf/messages.en:77]: As above

File: conf/messages.cy [hunk 2]
```diff
@@ -69,7 +69,8 @@ external.error.500.list-item2=Too many users on the service
 
 internal.error.500.title=There is a problem with the service
 internal.error.500.heading=Sorry, there is a problem with the service
-internal.error.500.message=Contact the Helpline.
+internal.error.500.line1=Try again later.
```
Comment (Tasmine10ten) [inline conf/messages.cy:72]: Title (line 70) will need to match heading to include 'Sorry'

Comment (slashrealm) [reply conf/messages.cy:72]: Good point. I missed that

Comment (chandaksweta) [reply conf/messages.cy:72]: sure thing.. will change

**Pull request merged**